src/main/java/seedu/address/MainApp.java

```diff
@@ -18,22 +18,22 @@
 import seedu.address.commons.exceptions.DataConversionException;
 import seedu.address.commons.util.ConfigUtil;
 import seedu.address.commons.util.StringUtil;
-import seedu.address.logic.AnakinLogic;
-import seedu.address.logic.AnakinLogicManager;
+import seedu.address.logic.Logic;
+import seedu.address.logic.LogicManager;
 import seedu.address.model.Anakin;
-import seedu.address.model.AnakinModel;
-import seedu.address.model.AnakinModelManager;
-import seedu.address.model.AnakinReadOnlyAnakin;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.ReadOnlyAnakin;
 import seedu.address.model.UserPrefs;
-import seedu.address.model.util.AnakinSampleDataUtil;
+import seedu.address.model.util.SampleDataUtil;
 import seedu.address.storage.AnakinStorage;
+import seedu.address.storage.JsonUserPrefsStorage;
 import seedu.address.storage.Storage;
 import seedu.address.storage.StorageManager;
-import seedu.address.storage.XmlAnakinStorage;
-import seedu.address.storage.JsonUserPrefsStorage;
 import seedu.address.storage.UserPrefsStorage;
+import seedu.address.storage.XmlStorage;
 import seedu.address.ui.Ui;
-import seedu.address.ui.UiManager;
+import seedu.address.ui.UiManagerAddressbook;
```

Comment:
`UiManager` looks like it already implements anakin's `UI`, why rename it to `UiManagerAddressBook`?